domain/schema/changelog.go

```diff
@@ -69,6 +69,7 @@ CREATE TABLE change_log_witness (
 // change_log table.
 func changeLogTriggersForTable(table, columnName string, namespaceID tableNamespaceID) func() schema.Patch {
 	return func() schema.Patch {
+		// TODO: Should we record the `NEW.*` instead of `OLD.*` for the update trigger?
```

Comment:
It doesn't matter, as it's an identifier for the entity and should not change.
We can remove this comment.